logging/logging.go

```diff
@@ -84,79 +91,41 @@ func SetLogLevel(level int) {
 	}
 }
 
-// GetLogLevel gets logrus log level and returns int Acra log level
-func GetLogLevel() int {
-	if log.GetLevel() == log.DebugLevel {
-		return LogDebug
-	}
-	if log.GetLevel() == log.InfoLevel {
-		return LogVerbose
+// CreateFormatter creates formatter object
+func CreateFormatter(format string) FormatterWrapper {
+	var formatter FormatterWrapper
+	switch strings.ToLower(format) {
+	case JsonFormatString:
+		formatter = JSONFormatter()
+	case CefFormatString:
+		formatter = CEFFormatter()
+	default:
+		formatter = TextFormatter()
 	}
-	return LogDiscard
-}
-
-// CustomizeBuilder allows to customize logging process
-type CustomizeBuilder struct {
-	writer        io.Writer
-	serviceName   string
-	loggingFormat string
-	hooks         []FormatterHook
-}
-
-// Customize is a global function for logging customization.
-// Example of usage: Customize().SetServiceName(...).SetFormat(...).SetOutput(...).Complete()
-func Customize() *CustomizeBuilder {
-	return &CustomizeBuilder{}
+	log.SetFormatter(formatter)
+	return formatter
 }
 
-// SetOutput specifies where logs should be written (stderr, file, etc.)
-func (c *CustomizeBuilder) SetOutput(w io.Writer) *CustomizeBuilder {
-	c.writer = w
-	return c
+// SetServiceName adds service-name label to log entries
+// (plaintext formatter ignores it)
+func SetServiceName(formatter FormatterWrapper, serviceName string) {
```

Comment:
why not add method `SetServiceName` to formatter object? Why we use OOP with functions and explicit first argument to object?)

Reply:
The reason is just stylistic. Previously we used to apply customizations like `log.SetOutput()`, `logging.SetServiceName()`, etc. So I decided to keep this style.

Reply:
Added https://github.com/cossacklabs/acra/pull/370/commits/4247aed0c2c46a91927634a72af3928a45511183 after @ilammy 's explanation https://github.com/cossacklabs/acra-enterprise/pull/41#discussion_r400994241